thirdeye/thirdeye-pinot/src/test/resources/com/linkedin/thirdeye/detection/yaml/alertconfig/alert-config-2.yaml

```diff
@@ -0,0 +1,42 @@
+subscriptionGroupName: "Subscription Group Name"
+cron: "0 0/5 * 1/1 * ? *"
+application: "test_application"
+active: true
+fromAddress: thirdeye@thirdeye.com
+
+type: DIMENSIONAL_ALERTER_PIPELINE
+dimensionRecipients:
+ "android":
+  - "android-thirdeye@thirdeye.com"
+ "ios":
+  - "ios-thirdeye@thirdeye.com"
+dimension: app_name
+
+detectionConfigIds:
+ - 5773069
```

Comment:
Is there a unique name to identify the detection config? Using the name would be more user friendly instead of id. We can do the conversion in backend.